src/Server/Services/Http/CrdCrudControllerBase.cs

```diff
@@ -221,7 +231,33 @@ private object CreateCrdFromType(T item)
             throw new ApplicationException($"Unsupported data type: {item.GetType()}");
         }
 
-        
+        private void CheckAeTitleAndName(DestinationApplicationEntity destAe)
+        {
+            Guard.Again.Null(destAe, nameof(destAe));
+
+            if (string.IsNullOrWhiteSpace(destAe.Name) && destAe.AeTitle.Contains("_"))
+            {
+                throw new Exception("Please specified the `name` for the DICOM destination when AE Title contains underscore (_).");
+            }
+            else if (destAe.Name.Contains("_"))
+            {
+                throw new Exception("The underscore character (_) is not allowed in the `name` field.");
+            }
+        }
+
+        private void CheckAeTitleAndName(ClaraApplicationEntity claraAe)
```

Comment:
What's the difference between this method and the one above? They looks exactly the same

Reply:
Oops.

Reply:
Different type.